src/ServiceInsight/app.config

```diff
@@ -36,4 +36,7 @@
       </dependentAssembly>
     </assemblyBinding>
   </runtime>
+	<appSettings>
+		<add key="SkipCertificateValidation" value="False" />
+	</appSettings>
```

Comment:
These lines are using tabs. They should be converted to spaces.

Reply:
Done